### ISSUE TITLE
JAWS Not Reading Button Labels in VSCode 1.66 and later

### ISSUE THREAD

**Issue by zuhairmahd** [FreedomScientific/VFO-standards-support#616]
### Summary

> Brief description of the issue. Please list any specific steps.
> 
> Example:
> 1. Go to http://www.google.com
> 2. Shift-Tab back to the About link
1. Install and open VSCode 1.66 or later.
2. Use the tab key to navigate to the toolbar, then use left and right arrow keys to navigate accross.  Buttons will be spoken without labels.  This also happens in the JSON editor, amongst other areas.
### Expected result
> What did you expect to happen?
JAWS should read the label of the toolbar button
### Actual result
> What actually did happen?
JAWS only says "button" for each element, failing to read the label
### Example

> A test case that demonstrates the issue - provide linked test case, CodePen, JSbin, etc...

#### JAWS version and build number
JAWS Version 2022.2202.38 ILM 

 
#### Operating System and version
Windows 11 Pro 21H2 (OS Build 22000.593)

#### Browser and version:
Microsoft Edge Version 100.0.1185.29 Official build 64-bit

**Comment by zuhairmahd** [comment]
[This issue in the VSCode repo ](https://github.com/microsoft/vscode/issues/14665
8)contains more context and contributions from @ggordon-vespero

**Comment by zuhairmahd** [comment]
Resolved by Microsoft in the next release
